sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -283,11 +338,13 @@ static rom_error_t mask_rom_boot(const manifest_t *manifest,
   flash_ctrl_exec_set(flash_exec);
   SEC_MMIO_WRITE_INCREMENT(kFlashCtrlSecMmioExecSet);
 
-  sec_mmio_check_values(rnd_uint32());
-  sec_mmio_check_counters(/*expected_check_count=*/4);
+  CFI_FUNC_COUNTER_PREPCALL(rom_counters, kCfiRomBoot, 2, kCfiRomPreBootCheck);
+  mask_rom_pre_boot_check();
```

Comment:
One last nit: It would be preferable before `flash_ctrl_exec_set` .

Reply:
Good point, done.